charge.go

```diff
@@ -355,6 +356,36 @@ type ChargePaymentMethodDetailsIdeal struct {
 	VerifiedName string `json:"verified_name"`
 }
 
+// ChargePaymentMethodDetailsKlarna represents details for the Klarna
+// PaymentMethod.
+type ChargePaymentMethodDetailsKlarna struct {
```

Comment:
I think there's a confusion here, there's nothing in PMD on Charge. In stripe-java it's currently empty: https://github.com/stripe/stripe-java/blob/master/src/main/java/com/stripe/model/Charge.java#L1570